migrate.js

```diff
@@ -0,0 +1,34 @@
+const admin = require("firebase-admin");
```

Comment:
This script is obviously not getting checked in any way or doing anything fancy with Babel...might be worth discussing, since future migrations will hopefully follow a similar pattern.